### PR TITLE
Efrs 1275 UI colors and sizing update fixes

### PR DESCRIPTION
**Pull request by smchedlidze826**

### CODE REVIEW

File: ui/src/app/features/model-statistics/model-statistics.component.ts [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    },
+  ];
+
+  ngOnInit(): void {
+    this.labels = this.type === 'month' ? this.getMonth(Object.values(this.statistics)) : this.getDays(Object.values(this.statistics));
+  }
+
+  getMonth(stats): Label[] {
+    let monthNames = [
+      'January',
```
Comment (isobolev) [inline ui/src/app/features/model-statistics/model-statistics.component.ts:54]: should come from translations file?

**Review by isobolev — COMMENTED** [review]
LGTM, couple minor comments

File: ui/src/app/features/application-collection-container/application-collection/application-collection.component.scss [hunk 2]
```diff
@@ -22,12 +22,11 @@
...
   align-items: center;
   flex-direction: column;
-  border-radius: 20px;
-  background: white;
-  width: 250px;
-  height: 200px;
-  margin: 1rem;
-  box-shadow: 1px 1px 1px $light-gray;
+  border-radius: 30px;
+  background: #eef1f5;
```
[the remark this answers is not in the document]
Comment (smchedlidze826) [reply ui/src/app/features/application-collection-container/application-collection/application-collection.component.scss:26]: fixed

File: ui/src/app/features/model-statistics/model-statistics.component.ts (hunk 1 above)
Comment (smchedlidze826) [reply ui/src/app/features/model-statistics/model-statistics.component.ts]: fixed

**Review by isobolev — APPROVED** [review]

**Pull request merged**